.travis.yml

```diff
@@ -25,6 +27,7 @@ script:
     fi
 cache: apt
 before_install:
+  - sudo apt-get update && sudo apt-get --reinstall install -qq language-pack-en language-pack-de language-pack-fr
```

Comment:
@rylnd,
yes.
In fact only the english and one non english language pack are needed, I will remove one.